pkg/webhook/sink_webhook.go

```diff
@@ -124,6 +124,11 @@ func (webhook *SinkWebhook) Default(ctx context.Context, obj runtime.Object) err
 	if r.Spec.Input.TypeClassName == "" {
 		r.Spec.Input.TypeClassName = "[B"
 	}
+
+	if r.Spec.LogTopicAgent == "" {
+		r.Spec.LogTopicAgent = v1alpha1.RUNTIME
```

Comment:
as discussed, for connectors we should use `SIDECAR` since the pulsar doesn't support log-topic for connectors natively

Reply:
@nlu90  updated